scripts/linters/python_linter.py

```diff
@@ -417,23 +377,29 @@ def _lint_py_files(self, config_pylint, config_pycodestyle):
                 python_utils.PRINT('Linting Python files %s to %s...' % (
                     current_batch_start_index + 1, current_batch_end_index))
 
-            with linter_utils.redirect_stdout(stdout):
+            with linter_utils.redirect_stdout(sys.stdout):
                 # This line invokes Pylint and prints its output
                 # to the target stdout.
                 pylint_report = StringMessageStream()
                 pylinter = lint.Run(
                     current_files_to_lint + [config_pylint],
                     reporter=text.TextReporter(pylint_report),
                     exit=False).linter
+
+            with linter_utils.redirect_stdout(stdout):
                 # These lines invoke Pycodestyle and print its output
                 # to the target stdout.
                 style_guide = pycodestyle.StyleGuide(
                     config_file=config_pycodestyle)
                 pycodestyle_report = style_guide.check_files(
                     paths=current_files_to_lint)
 
-            if pylinter.msg_status != 0 or pycodestyle_report.get_count() != 0:
+            if pycodestyle_report.get_count() != 0:
                 summary_message = stdout.getvalue()
+                summary_messages.append(summary_message)
+                are_there_errors = True
+
+            if pylinter.msg_status != 0:
```

Comment:
We need this here because we are not linting all files at once but in the batch of 50 files. So we need to store the result of those 50 files every time.

Reply:
still, we can move it around line 388, right?

Reply:
No, then it will read pylint output as well which is not what we want.